tox.ini

```diff
@@ -123,12 +159,33 @@ commands =
     pip install . {[testenv]build_single_wheel} --force-reinstall
     # install metatensor-operations
     pip install python/metatensor-operations {[testenv]build_single_wheel} --force-reinstall
+    # install metatensor-learn
+    pip install python/metatensor-learn {[testenv]build_single_wheel} --force-reinstall
     # install metatensor-torch
     pip install python/metatensor-torch {[testenv]build_single_wheel} --force-reinstall
 
     # run documentation tests
     pytest --doctest-modules --pyargs metatensor
 
+    # PR COMMENT technically to use the correct dependenicies for metatensor-learn
+    #            it should look like the commands below, but it does not work
+    #            because the metatensor-torch loads metatensor-learn in the learn.py
```

Comment:
The general problem is that metatensor-learn is dependent on a metatensor-operations release version and not on the most recent version in the master branch. So we need to split up the metatensor-torch tests and doctests tox environments into two environments, or install within the environments the metatensor-operations twice with different version. I am not sure which way to go. I feel like separating them is less error prone because it is structurally simpler, but the CI time increases.

Reply:
> The general problem is that metatensor-learn is dependent on a metatensor-operations release version and not on the most recent version in the master branch. 

Why?

It should depend on the git version IMO. Then whenever we do an breaking release of `metatensor-operations`, we also do the corresponding release for `-learn`.

Reply:
Yes I think you are right. That enforces to be always compatible with the latest `metatensor-operations` version with the latest `m̀etatensor-learn` release